package.json

```diff
@@ -63,9 +63,6 @@
     "sw-precache": "^5.2.0",
     "uglify-es": "^3.3.4"
   },
-  "engines": {
-    "node": "12"
```

Comment:
...unrelated dev helper: I can't get node 12 working on my Mac 😭 

`npx npm@6 install` did work to install.